### DISCUSSION TITLE
Please add simple and quick instructions for using HTTP/3 when publishing websites on Azure

### DISCUSSION THREAD

**Discussion by SetTrend** [microsoft/msquic#2291] [Q&A] [answered]
I'm missing quick instructions on how to set-up web services on Azure to host HTTP/3.

If I, for instance, was to publish a .NET Core Web API to Azure, or any NodeJS front-end, like Angular … how would I do that to have my services use HTTP/3?

**Comment by nibanks** [comment, marked as answer]
Unfortunately, this repo (and the folks who work on it) is simply for the QUIC layers, and not HTTP/3. If you are interested in .NET HTTP/3 usage, I suggest you start with the .NET repo. cc @wfurt @manickap @samsp-msft

**Reply by SetTrend** [reply]
Please pardon me for addressing the wrong group of people in this repo.

I believed that QUIC was the thing to install/enable in, e.g. IIS, to automatically get HTTP/3 support.


Unfortunately, documentation is quite sparse on QUIC on Windows yet.

Do you want me to close this discussion?

**Reply by ManickaP** [reply]
HTTP/3 is supported with Kestrel, how to set it up is here: https://docs.microsoft.com/en-us/aspnet/core/fundamentals/servers/kestrel/http3?view=aspnetcore-6.0
AFAIK http.sys has H/3 support as well, more details are in this article: https://devblogs.microsoft.com/dotnet/http-3-support-in-dotnet-6/

Does this cover your scenario?

**Reply by SetTrend** [reply]
Ah, yes. Thanks a lot for this valuable information! 👍👍👍
